test/functional/mining_getblocktemplate_longpoll.py

```diff
@@ -29,45 +31,48 @@ def set_test_params(self):
         self.num_nodes = 2
         self.supports_cli = False
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
-        self.log.info("Warning: this test will take about 70 seconds in the best case. Be patient.")
-        self.nodes[0].generate(10)
+        self.log.info("Warning: this test will take about 70 seconds in the worst case. Be patient.")
```

Comment:
why this change?

Reply:
Hm, I had a few test runs that finished in ~6 seconds (i.e. introduction of a new transaction would terminate the longpoll immediately, not only in +1minute), but can't reproduce it anymore. Changed the message back to original.